toucan_connectors/snowflake_oauth2/snowflake_oauth2_connector.py

```diff
@@ -21,6 +21,60 @@ class SnowflakeoAuth2DataSource(SnowflakeDataSource):
 
 
 class SnowflakeoAuth2Connector(ToucanConnector):
+    _auth_flow = 'oauth2'
+    _oauth_trigger = 'connector'
+    data_source_model: SnowflakeoAuth2DataSource
+
+    token_url: str = Field(None, **{'hidden': True})
+    auth_flow_id: str = Field(None, **{'hidden': True})
+    oauth2_version = Field('1', **{'hidden': True})
+    authorization_url: str = Field(None, **{'hidden': True})
+    redirect_uri: str = Field(None, **{'hidden': True})
+    category: Category = Field(Category.SNOWFLAKE, **{'hidden': True})
+
+    info_step1: str = Field(
+        '<div style="width: 100%; padding: 10px; background-color: #2a66a1;">Step 1<br />Please fill connector name</div>',
+        title='step_1',
+        widget='info',
+    )
+
+    info_step2: str = Field(
+        '<div style="width: 100%; padding: 10px; background-color: #2a66a1;">Step 2<br />Play this request in Snowflake<br />'
+        'create security integration toucan_oauth2_{{name}}<br />'
+        '<span style="color: red;">'
+        'type = oauth<br />'
+        'enabled = true<br />'
+        'oauth_client = custom<br />'
+        'oauth_client_type = "CONFIDENTIAL"<br />'
+        'oauth_redirect_uri = "https://localhost:5000/{{small_app_name}}/oauth/redirect?connector_name={{name}}"<br />'
```

Comment:
```suggestion
        'oauth_redirect_uri = "https://{{backend}}/{{small_app_name}}/oauth/redirect?connector_name={{name}}"<br />'
```
Ici il faudrait afficher dynamiquement le nom de l'instance. On peut le récupérer pareillement que le small_app_name.

Reply:
On est d'accord
Mais il ne faut pas les prendre en paramètre de la méthode, le mieux serait potentiellement de les récupérer de la même manière que le _connector_name_ pour ne pas renforcer le lien entre les différentes repos

Reply:
Oui très bien.